builtin/staker/validations_test.go

```diff
@@ -385,68 +351,59 @@ func TestStaker_QueueUpValidators(t *testing.T) {
 	addr4 := datagen.RandAddress()
 
 	stake := RandomStake()
-	err := staker.AddValidator(addr1, addr1, uint32(360)*24*15, stake)
-	assert.NoError(t, err)
 
-	validator, err := staker.Get(addr1)
-	assert.NoError(t, err)
-	assert.False(t, validator.IsEmpty())
-	assert.Equal(t, stake, validator.PendingLocked)
-	assert.Equal(t, validation.StatusQueued, validator.Status)
+	NewSequence(staker).
+		AddValidator(addr1, addr1, uint32(360)*24*15, stake).
```

Comment:
```suggestion
		AddValidator(addr1, addr1, MediumStakingPeriod, stake).
```
I saw you've used the `MediumStakingPeriod` variable in another test case, I think we can reuse it here

Reply:
Will append these changes to number 4 @paologalligit, to avoid merge conflicts

Reply:
https://github.com/vechain/thor/pull/1209/files#diff-b64b0832284fa152124ed3b2676ef678e327bc0be1b9edc8d354bf654c5cfd2b